knotx-core/pom.xml

```diff
@@ -100,6 +100,15 @@
       <groupId>uk.co.datumedge</groupId>
       <artifactId>hamcrest-json</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-all</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
```

Comment:
Mockito has `provided` scope in *knotx-junit* module.
It looks like in core module the default scope could be used (test).
Or am I wrong? Could we remove thes scope override here?
(should have)